lms/envs/test.py

```diff
@@ -551,6 +551,10 @@
 
 LMS_ROOT_URL = "http://localhost:8000"
 
+# TODO (felipemontoya): This key is only needed during lettuce tests.
+# To be removed during https://openedx.atlassian.net/browse/DEPR-19
+FRONTEND_LOGOUT_URL = LMS_ROOT_URL + '/logout'
```

Comment:
Question: do the lettuce tests not pass with this living in cms/envs/test.py?

Reply:
Really good question. I did not know either, so I tried, and the answer is no (https://build.testeng.edx.org/job/edx-platform-lettuce-pipeline-pr/2264/console).
I'm returning to the working version

Reply:
Thanks for trying @felipemontoya.